template/src/ontology/run.sh.jinja2

```diff
@@ -63,7 +63,14 @@ if [ -n "$TAG_IN_IMAGE" ]; then
   ODK_IMAGE=$(echo $ODK_IMAGE | awk -F':' '{ print $1 }')
 fi
 ODK_TAG=${ODK_TAG:-latest}
-ODK_JAVA_OPTS={% if project.robot_java_args is defined and project.robot_java_args|length %}${ODK_JAVA_OPTS:-{{ project.robot_java_args }}}{% else %}${ODK_JAVA_OPTS:--Xmx8G}{% endif %}
+{% if project.robot_java_args is defined and project.robot_java_args|length %}
```

Comment:
Never realised that `project.robot_java_args|length` condition is necessary to catch the case that someone sets the key to an empty value.. Makes sense though.